pkgs/development/libraries/cxxopts/default.nix

```diff
@@ -12,16 +12,21 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = lib.optional enableUnicodeHelp [ icu.dev ];
-  cmakeFlags = lib.optional enableUnicodeHelp [ "-DCXXOPTS_USE_UNICODE_HELP=TRUE" ];
+  cmakeFlags = [ "-DCXXOPTS_BUILD_EXAMPLES=OFF" ]
+    ++ lib.optional enableUnicodeHelp "-DCXXOPTS_USE_UNICODE_HELP=TRUE"
+    # Due to -Wsuggest-override, remove when cxxopts is updated
+    ++ lib.optional stdenv.isDarwin "-DCXXOPTS_ENABLE_WARNINGS=OFF";
   nativeBuildInputs = [ cmake ] ++ lib.optional enableUnicodeHelp [ pkg-config ];
 
   doCheck = true;
 
+  # Conflict on case-insensitive filesystems.
+  dontUseCmakeBuildDir = true;
+
   meta = with lib; {
     homepage = "https://github.com/jarro2783/cxxopts";
     description = "Lightweight C++ GNU-style option parser library";
     license = licenses.mit;
     maintainers = [ maintainers.spease ];
-    platforms = platforms.all;
```

Comment:
Why did you remove this?

Reply:
I was asked to remove this in another review. Assumed it meant the same when nothing was specified?

Regardless, it probably shouldn't be in this PR. I'll remove it.

Reply:
probably because it's the default. But i don't think it detracts from readability since meta is at the bottom of the file anyway.